tests/api/api/test_utils.py

```diff
@@ -1032,6 +1033,28 @@ def test_ensure_function_security_context_unknown_enrichment_mode(
     )
 
 
+def test_ensure_function_security_context_missing_control_plane(
```

Comment:
Please add 2 more tests:
* Where `user_unix_id` is not `None` and you assert that `get_user_unix_id` isn't called, but there's no exception either.
* Where `user_unix_id` is `None` and  you assert that `get_user_unix_id` is called, and there's no exception either.

Reply:
added (1) as part of an existing test
added new test for (2)